app/config/collections.php

```diff
@@ -249,11 +296,11 @@
         ],
         'indexes' => [
             [
-                '$id' => '_key_collection',
+                '$id' => '_key_db_collection',
                 'type' => Database::INDEX_KEY,
-                'attributes' => ['collectionId'],
-                'lengths' => [Database::LENGTH_KEY],
-                'orders' => [Database::ORDER_ASC],
+                'attributes' => ['collectionId', 'databaseId'],
```

Comment:
If we there is a query of databaseId without collectionId so better ['databaseId', 'collectionId']